src/pages/Dashboard/Dashboard.scss

```diff
@@ -4,7 +4,7 @@
   margin-bottom: 20px;
 }
 
-@media screen and (max-width: 800px) {
+@media screen and (max-width: 850px) {
```

Comment:
Why did you use these media instead of those specified in the variables??? Look at file "_variables.scss".